ui/src/containers/VolumePage.js

```diff
@@ -76,6 +76,7 @@ const VolumePage = (props) => {
   const node = useSelector((state) => makeGetNodeFromUrl(state, props));
   const nodes = useSelector((state) => state.app.nodes.list);
   const volumes = useSelector((state) => state.app.volumes.list);
+  const volumesLoading = useSelector((state) => state.app.volumes.loading);
```

Comment:
We can't arrive at the `EmptyState Page` in Volume Page, because we use `isLoading` property instead of `loading` ;)

Should be: 
```js
const volumesLoading = useSelector((state) => state.app.volumes.isLoading);
```